cachito/workers/pkg_managers/general_js.py

```diff
@@ -451,3 +462,30 @@ def upload_non_registry_dependency(dep_identifier, version_suffix, verify_script
 
         repo_name = get_js_hosted_repo_name()
         nexus.upload_artifact(repo_name, "npm", modified_dep_archive)
+
+
+def _verify_checksum(file_path, checksum_info, chunk_size=10240):
+    """
+    Verify the checksum of the file at the given path matches the expected checksum info.
+
+    :param str file_path: the path to the file to be verified
+    :param ChecksumInfo checksum_info: the expected checksum information
```

Comment:
Missing param `chunk_size` from the doc string

Reply:
Good catch! Fixed it.